charts/hono/values.yaml

```diff
@@ -1001,6 +1003,68 @@ deviceRegistryExample:
     # The password to use for authenticating to the MongoDB instance.
     # password:
 
+  # jdbcBasedDeviceRegistry contains configuration properties specific to the
+  # jdbc based device registry.
+  jdbcBasedDeviceRegistry:
+    # imageName contains the name (excluding registry)
+    # of the container image for the example jdbc based device registry.
+    imageName: eclipse/hono-service-device-registry-jdbc
+    # imageTag contains the tag of the container image to deploy.
+    # If not specified, the value of the honoImagesTag property is used.
+    # imageTag:
+    # containerRegistry contains the name of the container registry to pull
+    # the image from.
+    # If not specified, the value of the "honoContainerRegistry" property is used.
+    # containerRegistry:
+    # registry.jdbc contains the configuration properties for device registry
+    # to connect to the database.
+    registry:
+      jdbc:
+        # the jdbc properties for read-only operations
+        adapter:
+          # The jdbc URL for the database
+          url:
+          # The jdbc driver class name
+          driverClass:
+          # The user name to use for authenticating to the database
+          username:
+          # The password to use for authenticating to the database
+          password:
+        # the jdbc properties for write operations
+        management:
+          # The jdbc URL for the database. For SQLServer, ";SelectMethod=Cursor" is required in the management url.
+          url:
+          # The jdbc driver class name
+          driverClass:
+          # The user name to use for authenticating to the database
+          username:
+          # The password to use for authenticating to the database
+          password:
+    # tenant.jdbc contains the configuration properties for tenant registry
+    # to connect to the database.
+    tenant:
+      jdbc:
+        # the jdbc properties for read-only operations
+        adapter:
+          # The jdbc URL for the database
+          url:
+          # The jdbc driver class name
+          driverClass:
+          # The user name to use for authenticating to the database
+          username:
+          # The password to use for authenticating to the database
+          password:
+        # the jdbc properties for write operations
+        management:
+          # The jdbc URL for the database. For SQLServer, ";SelectMethod=Cursor" is required in the management url.
+          url:
+          # The jdbc driver class name
+          driverClass:
+          # The user name to use for authenticating to the database
+          username:
+          # The password to use for authenticating to the database
+          password:
+
```

Comment:
it would be nice to have default configuration for using an in memory H2 DB so that you can simply start the registry and use it, without the need to set a lot of configuration properties ...

Reply:
point taken, however setting up h2 using a persistent file would require quite some changes to the helm chart. Would in-memory h2 be ok?

Reply:
sure, we can still evolve from there ...

Reply:
I am not sure I get the issue with persistence. That would be one `pvc` and a volume mapping in case of `h2`. Could be a simple feature flag to enable that.